dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java

```diff
@@ -872,17 +881,24 @@ protected void handleList() {
         }
     }
 
+    protected List<String> getClientSpecificCommands() {
```

Comment:
Gets commands that only apply to the WorkflowClient for the suggestion tool.

WES only applies to workflow tool, but it is dealt with [here](https://github.com/dockstore/dockstore-cli/blob/a11fec6c157b885281abe8e3eda7185f683bd156/dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java#L234-L237) and [here](https://github.com/dockstore/dockstore-cli/blob/a11fec6c157b885281abe8e3eda7185f683bd156/dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java#L323-L330). Should I create a ticket to move it to WorkflowClient.